FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -5030,7 +5030,7 @@ OrderedNode *OpDispatchBuilder::LoadSource_WithOpSize(FEXCore::IR::RegisterClass
       // Now extract the subregister if it was a partial load /smaller/ than SSE size
       // TODO: Instead of doing the VMov implicitly on load, hunt down all use cases that require partial loads and do it after load.
```

Comment:
drop the todo maybe?

Reply:
I'd say the todo is still relevant because there is code still doing vmovs when they don't need to. Particularly relevant to AVX code that is doing it.